minikerberos/communication.py

```diff
@@ -225,7 +220,7 @@ def get_TGT(self, override_etype = None, decrypt_tgt = True):
 			3. PROFIT
 		"""
 		logger.debug('Generating initial TGT without authentication data')
-		now = datetime.datetime.utcnow()
+		now = datetime.datetime.now(datetime.timezone.utc)
```

Comment:
If this timezone specification is there, then why freezing the asn1crypto to a specific version?

Reply:
I had problems with the latest asn1crypto. Some API change. That is why I fixed it to a working version.

Reply:
yes, you are right, but I decided to update the code to support the new version instead :) I needed to update the time fields to discard microsecond info as it's not supported by the kerberos standard